Sources/StreamChatUI/ChatMessageList/ChatMessageListVC.swift

```diff
@@ -563,7 +563,6 @@ open class ChatMessageListVC:
     
     open func messageContentViewDidTapOnQuotedMessage(_ indexPath: IndexPath?) {
         guard let indexPath = indexPath else { return log.error("IndexPath is not available") }
-        print(#function, indexPath)
```

Comment:
Should this be exchanged with other logger?
like log.info or log.debug

Reply:
We are not printing anything useful so `log` is not needed IMO. 

But in that case, we may need to delete the above statement where we get indexPath as it will generate an unused warning. Let's replace the print with `log.info()` 👍🏻

Reply:
I think it would be useful to log something like: "You tapped on a quoted message, you can implement this method if you want to do something about it"

Reply:
This looks more like a debugging leftover than reasonable logging. Might be wrong, though